Core/src/org/sleuthkit/autopsy/centralrepository/ingestmodule/IngestModule.java

```diff
@@ -195,6 +197,7 @@ public void shutDown() {
     })
     @Override
     public void startUp(IngestJobContext context) throws IngestModuleException {
+        IngestEventsListener.enableCentralRepositoryModule();
```

Comment:
I believe that this method should be called at the very end of startUp, not the beginning. There are a lot of places in the code that follows where an IngestModuleException is thrown, meaning that the module will not run and the ingest job will not be started.

Reply:
I was assuming shutdown would still be called if startup threw an exception but based on your comment I am gathering that is incorrect, and I will adjust it to reflect that since my placement of the calls as the first thing in both methods was intended to ensure both were always executed when the module was selected.

Reply:
I tested and shutdown is still called if startup threw an exception, so in order to keep the count accurate if I move the method calls to the end of startup and shutdown I need to first ensure that shutdown will also detect the same issues that startup detects prior to the method call.

Reply:
I feel like given the current error try catch structure of shutDown and startUp that the farthest down I can safely move the call in startUp is Line 236  of startUp after the dbManager instance is gotten but before the filesType variable is set.   With the call in startUp at that location the call in shutDown would need to occur after line 170.